### PR TITLE
egglib: add aarch64/arm64 builds

### PR DESCRIPTION
**Pull request by martin-g**
Describe your pull request here

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

### General instructions

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

### Instructions for avoiding API, ABI, and CLI breakage issues
Conda is able to record and lock (a.k.a. pin) dependency versions used at build time of other recipes.
This way, one can avoid that expectations of a downstream recipe with regards to API, ABI, or CLI are violated by later changes in the recipe.
If not already present in the meta.yaml, make sure to specify `run_exports` (see [here](https://bioconda.github.io/contributor/linting.html#missing-run-exports) for the rationale and comprehensive explanation).
Add a `run_exports` section like this:

```yaml
build:
  run_exports:
    - ...

```

with `...` being one of:

| Case                             | run_exports statement                                               |
| -------------------------------- | ------------------------------------------------------------------- |
| semantic versioning              | `{{ pin_subpackage("myrecipe", max_pin="x") }}`     |
| semantic versioning (0.x.x)      | `{{ pin_subpackage("myrecipe", max_pin="x.x") }}`   |
| known breakage in minor versions | `{{ pin_subpackage("myrecipe", max_pin="x.x") }}` (in such a case, please add a note that shortly mentions your evidence for that) |
| known breakage in patch versions | `{{ pin_subpackage("myrecipe", max_pin="x.x.x") }}` (in such a case, please add a note that shortly mentions your evidence for that) |
| calendar versioning              | `{{ pin_subpackage("myrecipe", max_pin=None) }}`    |

while replacing `"myrecipe"` with either `name` if a `name|lower` variable is defined in your recipe or with the lowercase name of the package in quotes.

### Bot commands for PR management

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

Note that the <code>@BiocondaBot please merge</code> command is now depreciated. Please just squash and merge instead.

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by martin-g** [comment]
@BiocondaBot please fetch artifacts

**Comment by martin-g** [comment]
```

mgrigorov in 🌐 euler-arm-22 in /tmp/egglib via 🅒 test123 took 25s 
❯ tar xvf egglib-3.3.3-py310hd7577d4_1.tar.bz2                                                                                                                                                             (test123) 
info/run_exports.json
info/hash_input.json
info/test/run_test.py
info/index.json
info/has_prefix
info/recipe/meta.yaml.template
info/recipe/meta.yaml
info/recipe/conda_build_config.yaml
info/about.json
info/files
info/licenses/LICENSE
info/paths.json
info/git
lib/python3.10/site-packages/egglib/test/data/example_E3.fg
lib/python3.10/site-packages/egglib/test/data/example_E2.fg
lib/python3.10/site-packages/egglib/test/data/example.fg
lib/python3.10/site-packages/egglib/test/data/example_E.fg
lib/python3.10/site-packages/egglib/test/data/clustal_error8.aln
lib/python3.10/site-packages/egglib/test/data/clustal_error2.aln
lib/python3.10/site-packages/egglib/test/data/clustal_error7.aln
lib/python3.10/site-packages/egglib/test/data/clustal_error5.aln
lib/python3.10/site-packages/egglib/test/data/clustal_error1.aln
lib/python3.10/site-packages/egglib/test/data/align.aln
lib/python3.10/site-packages/egglib/test/data/clustal_error6.aln
lib/python3.10/site-packages/egglib/test/data/clustal_error3.aln
lib/python3.10/site-packages/egglib/test/data/clustal_error4.aln
lib/python3.10/site-packages/egglib/test/data/LYK.prot.E.aln
lib/python3.10/site-packages/egglib/test/data/LYK.prot.aln
lib/python3.10/site-packages/egglib/test/data/human_fragment.vcfi
lib/python3.10/site-packages/egglib/cli/__pycache__/__init__.cpython-310.pyc
lib/python3.10/site-packages/egglib/test/__pycache__/__init__.cpython-310.pyc
lib/python3.10/site-packages/egglib/test/io/__pycache__/__init__.cpython-310.pyc
lib/python3.10/site-packages/egglib/test/base/__pycache__/__init__.cpython-310.pyc
lib/python3.10/site-packages/egglib/test/stats/__pycache__/__init__.cpython-310.pyc
lib/python3.10/site-packages/egglib/test/tools/__pycache__/__init__.cpython-310.pyc
lib/python3.10/site-packages/egglib/test/wrappers/__pycache__/__init__.cpython-310.pyc
lib/python3.10/site-packages/egglib/test/coalesce/__pycache__/__init__.cpython-310.pyc
lib/python3.10/site-packages/egglib/__pycache__/config.cpython-310.pyc
lib/python3.10/site-packages/egglib/coalesce/__pycache__/__init__.cpython-310.pyc
lib/python3.10/site-packages/egglib/stats/__pycache__/__init__.cpython-310.pyc
lib/python3.10/site-packages/egglib/test/io/__pycache__/test_index.cpython-310.pyc
lib/python3.10/site-packages/egglib/tools/__pycache__/__init__.cpython-310.pyc
lib/python3.10/site-packages/egglib/wrappers/__pycache__/__init__.cpython-310.pyc
lib/python3.10/site-packages/egglib/__pycache__/__init__.cpython-310.pyc
lib/python3.10/site-packages/egglib/io/__pycache__/__init__.cpython-310.pyc
lib/python3.10/site-packages/egglib/wrappers/__pycache__/_run.cpython-310.pyc
lib/python3.10/site-packages/egglib/__pycache__/eggwrapper.cpython-310.pyc
lib/python3.10/site-packages/egglib/tools/__pycache__/_to_codons.cpython-310.pyc
lib/python3.10/site-packages/egglib/io/__pycache__/_genepop.cpython-310.pyc
lib/python3.10/site-packages/egglib/test/wrappers/__pycache__/test_clustal.cpython-310.pyc
lib/python3.10/site-packages/egglib/test/stats/__pycache__/test_site_diversity.cpython-310.pyc
lib/python3.10/site-packages/egglib/test/stats/__pycache__/test_baudry.cpython-310.pyc
lib/python3.10/site-packages/egglib/test/tools/__pycache__/test_concat.cpython-310.pyc
lib/python3.10/site-packages/egglib/test/stats/__pycache__/test_ehh.cpython-310.pyc
lib/python3.10/site-packages/egglib/test/io/__pycache__/test_genepop.cpython-310.pyc
lib/python3.10/site-packages/egglib/test/base/__pycache__/test_site.cpython-310.pyc
lib/python3.10/site-packages/egglib/test/wrappers/__pycache__/test_nj.cpython-310.pyc
lib/python3.10/site-packages/egglib/test/io/__pycache__/test_legacy.cpython-310.pyc
lib/python3.10/site-packages/egglib/test/io/__pycache__/test_fasta.cpython-310.pyc
lib/python3.10/site-packages/egglib/test/tools/__pycache__/test_reading_frame.cpython-310.pyc
lib/python3.10/site-packages/egglib/test/coalesce/__pycache__/test_simulator.cpython-310.pyc
lib/python3.10/site-packages/egglib/test/stats/__pycache__/test_paralog_pi.cpython-310.pyc
lib/python3.10/site-packages/egglib/cli/__pycache__/test.cpython-310.pyc
lib/python3.10/site-packages/egglib/io/__pycache__/_ms.cpython-310.pyc
lib/python3.10/site-packages/egglib/wrappers/__pycache__/_nj.cpython-310.pyc
lib/python3.10/site-packages/egglib/cli/__pycache__/config.cpython-310.pyc
lib/python3.10/site-packages/egglib/stats/__pycache__/_baudry.cpython-310.pyc
lib/python3.10/site-packages/egglib/io/__pycache__/_fasta.cpython-310.pyc
lib/python3.10/site-packages/egglib/tools/__pycache__/_concat.cpython-310.pyc
lib/python3.10/site-packages/egglib/stats/__pycache__/_haplotypes.cpython-310.pyc
lib/python3.10/site-packages/egglib/test/stats/__pycache__/test_ld.cpython-310.pyc
lib/python3.10/site-packages/egglib/stats/__pycache__/_paralog_pi.cpython-310.pyc
lib/python3.10/site-packages/egglib/test/stats/__pycache__/test_haplotypes.cpython-310.pyc
lib/python3.10/site-packages/egglib/wrappers/__pycache__/_clustal.cpython-310.pyc
lib/python3.10/site-packages/egglib/test/stats/__pycache__/test_coding_diversity.cpython-310.pyc
lib/python3.10/site-packages/egglib/test/io/__pycache__/test_gff3.cpython-310.pyc
lib/python3.10/site-packages/egglib/tools/__pycache__/_reading_frame.cpython-310.pyc
lib/python3.10/site-packages/egglib/test/wrappers/__pycache__/test_muscle.cpython-310.pyc
lib/python3.10/site-packages/egglib/test/wrappers/__pycache__/test_phyml.cpython-310.pyc
lib/python3.10/site-packages/egglib/test/io/__pycache__/test_ms.cpython-310.pyc
lib/python3.10/site-packages/egglib/io/__pycache__/_legacy.cpython-310.pyc
lib/python3.10/site-packages/egglib/test/tools/__pycache__/test_seq_manip.cpython-310.pyc
lib/python3.10/site-packages/egglib/stats/__pycache__/_ld.cpython-310.pyc
lib/python3.10/site-packages/egglib/test/stats/__pycache__/test_freq.cpython-310.pyc
lib/python3.10/site-packages/egglib/test/tools/__pycache__/test_random.cpython-310.pyc
lib/python3.10/site-packages/egglib/__pycache__/_site.cpython-310.pyc
lib/python3.10/site-packages/egglib/test/wrappers/__pycache__/test_codeml.cpython-310.pyc
lib/python3.10/site-packages/egglib/wrappers/__pycache__/_utils.cpython-310.pyc
lib/python3.10/site-packages/egglib/stats/__pycache__/_coding_diversity.cpython-310.pyc
lib/python3.10/site-packages/egglib/__pycache__/alphabets.cpython-310.pyc
lib/python3.10/site-packages/egglib/wrappers/__pycache__/_muscle.cpython-310.pyc
lib/python3.10/site-packages/egglib/tools/__pycache__/_seq_manip.cpython-310.pyc
lib/python3.10/site-packages/egglib/test/io/__pycache__/test_vcf.cpython-310.pyc
lib/python3.10/site-packages/egglib/__pycache__/_freq.cpython-310.pyc
lib/python3.10/site-packages/egglib/stats/__pycache__/_ehh.cpython-310.pyc
lib/python3.10/site-packages/egglib/test/tools/__pycache__/test_code_tools.cpython-310.pyc
lib/python3.10/site-packages/egglib/wrappers/__pycache__/_phyml.cpython-310.pyc
lib/python3.10/site-packages/egglib/test/io/__pycache__/test_genbank.cpython-310.pyc
lib/python3.10/site-packages/egglib/coalesce/__pycache__/_simulator.cpython-310.pyc
lib/python3.10/site-packages/egglib/test/base/__pycache__/test_structure.cpython-310.pyc
lib/python3.10/site-packages/egglib/io/__pycache__/_gff3.cpython-310.pyc
lib/python3.10/site-packages/egglib/wrappers/__pycache__/_codeml.cpython-310.pyc
lib/python3.10/site-packages/egglib/test/base/__pycache__/test_tree.cpython-310.pyc
lib/python3.10/site-packages/egglib/io/__pycache__/_genbank.cpython-310.pyc
lib/python3.10/site-packages/egglib/test/coalesce/__pycache__/test_param_helpers.cpython-310.pyc
lib/python3.10/site-packages/egglib/tools/__pycache__/_code_tools.cpython-310.pyc
lib/python3.10/site-packages/egglib/stats/__pycache__/_cstats.cpython-310.pyc
lib/python3.10/site-packages/egglib/coalesce/__pycache__/_param_helpers.cpython-310.pyc
lib/python3.10/site-packages/egglib/io/__pycache__/_vcf.cpython-310.pyc
lib/python3.10/site-packages/egglib/test/base/__pycache__/test_alphabet.cpython-310.pyc
lib/python3.10/site-packages/egglib/test/io/__pycache__/test_vcf_legacy.cpython-310.pyc
lib/python3.10/site-packages/egglib/test/wrappers/__pycache__/test_blast.cpython-310.pyc
lib/python3.10/site-packages/egglib/__pycache__/_tree.cpython-310.pyc
lib/python3.10/site-packages/egglib/wrappers/__pycache__/_blast.cpython-310.pyc
lib/python3.10/site-packages/egglib/test/base/__pycache__/test_base.cpython-310.pyc
lib/python3.10/site-packages/egglib/test/stats/__pycache__/test_cstats.cpython-310.pyc
lib/python3.10/site-packages/egglib/__pycache__/_interface.cpython-310.pyc
lib/python3.10/site-packages/egglib/test/data/a.vcf
lib/python3.10/site-packages/egglib/test/data/b.vcf
lib/python3.10/site-packages/egglib/test/data/example1.vcf
lib/python3.10/site-packages/egglib/test/data/exemple1_F.vcf
lib/python3.10/site-packages/egglib/test/data/example_E2.vcf
lib/python3.10/site-packages/egglib/test/data/tmp_MFD75200_PL.vcf
lib/python3.10/site-packages/egglib/test/data/human_fragment.vcf
lib/python3.10/site-packages/egglib/test/data/tmp_HF_GL.vcf
lib/python3.10/site-packages/egglib/test/data/scaffold_285.vcf
lib/python3.10/site-packages/egglib/test/data/merged_filt_depth_75_200_ssduplicateindANDsnp-ssblanck_DEF_for_structure.vcf
lib/python3.10/site-packages/EggLib-3.3.3.dist-info/direct_url.json
lib/python3.10/site-packages/egglib/test/data/test001.fas
lib/python3.10/site-packages/egglib/test/data/align004.fas
lib/python3.10/site-packages/egglib/test/data/align003.fas
lib/python3.10/site-packages/egglib/test/data/align005.fas
lib/python3.10/site-packages/egglib/test/data/align006.fas
lib/python3.10/site-packages/egglib/test/data/example_Els.fas
lib/python3.10/site-packages/egglib/test/data/align007.fas
lib/python3.10/site-packages/egglib/test/data/align002.fas
lib/python3.10/site-packages/egglib/test/data/align001.fas
lib/python3.10/site-packages/egglib/test/data/simul002.fas
lib/python3.10/site-packages/egglib/test/data/triconfig.fas
lib/python3.10/site-packages/egglib/test/data/001-0006534.fas
lib/python3.10/site-packages/egglib/test/data/simul013.fas
lib/python3.10/site-packages/egglib/test/data/simul004.fas
lib/python3.10/site-packages/egglib/test/data/cds_e.fas
lib/python3.10/site-packages/egglib/test/data/simul005.fas
lib/python3.10/site-packages/egglib/test/data/simul014.fas
lib/python3.10/site-packages/egglib/test/data/simul-cds.fas
lib/python3.10/site-packages/egglib/test/data/c_file_1.1.fas
lib/python3.10/site-packages/egglib/test/data/c_file_2.fas
lib/python3.10/site-packages/egglib/test/data/c_file_1.fas
lib/python3.10/site-packages/egglib/test/data/align_generate.fas
lib/python3.10/site-packages/egglib/test/data/At-Ia.fas
lib/python3.10/site-packages/egglib/test/data/simul003.fas
lib/python3.10/site-packages/egglib/test/data/50_seqs_100_100_bp.fas
lib/python3.10/site-packages/egglib/test/data/cds.fas
lib/python3.10/site-packages/egglib/test/data/sequence.fas
lib/python3.10/site-packages/egglib/test/data/simul006.fas
lib/python3.10/site-packages/egglib/test/data/cds_clust.fas
lib/python3.10/site-packages/egglib/test/data/codon_align_cstop.fas
lib/python3.10/site-packages/egglib/test/data/codon_align.fas
lib/python3.10/site-packages/egglib/test/data/simul011.fas
lib/python3.10/site-packages/egglib/test/data/simul010.fas
lib/python3.10/site-packages/egglib/test/data/simul007.fas
lib/python3.10/site-packages/egglib/test/data/simul008.fas
lib/python3.10/site-packages/egglib/test/data/simul009.fas
lib/python3.10/site-packages/egglib/test/data/example.fas
lib/python3.10/site-packages/egglib/test/data/example_ename.fas
lib/python3.10/site-packages/egglib/test/data/example_edpl.fas
lib/python3.10/site-packages/egglib/test/data/prot_data.fas
lib/python3.10/site-packages/egglib/test/data/125_seqs_300_300_bp.fas
lib/python3.10/site-packages/egglib/test/data/simul012.fas
lib/python3.10/site-packages/egglib/test/data/simul001.fas
lib/python3.10/site-packages/egglib/test/data/align1_E.fas
lib/python3.10/site-packages/egglib/test/data/error.fas
lib/python3.10/site-packages/egglib/test/data/FTLa.fas
lib/python3.10/site-packages/egglib/test/data/align1_F.fas
lib/python3.10/site-packages/egglib/test/data/eIF4E.fas
lib/python3.10/site-packages/egglib/test/data/dmi1.fas
lib/python3.10/site-packages/egglib/test/data/dmi3.fas
lib/python3.10/site-packages/egglib/test/data/lax4.fas
lib/python3.10/site-packages/egglib/test/data/align1_T.fas
lib/python3.10/site-packages/egglib/test/data/nucl_data.fas
lib/python3.10/site-packages/EggLib-3.3.3.dist-info/top_level.txt
lib/python3.10/site-packages/EggLib-3.3.3.dist-info/entry_points.txt
lib/python3.10/site-packages/egglib/test/data/sites004.txt
lib/python3.10/site-packages/egglib/test/data/sites001.txt
lib/python3.10/site-packages/egglib/test/data/genepop2.txt
lib/python3.10/site-packages/egglib/test/data/genepop0.txt
lib/python3.10/site-packages/egglib/test/data/genepop1.txt
lib/python3.10/site-packages/egglib/test/data/genepop003.txt
lib/python3.10/site-packages/egglib/test/data/genepop002.txt
lib/python3.10/site-packages/egglib/test/data/sites002.txt
lib/python3.10/site-packages/egglib/test/data/sites003.txt
lib/python3.10/site-packages/egglib/test/data/genepop001.txt
lib/python3.10/site-packages/egglib/test/data/results_prot.txt
lib/python3.10/site-packages/egglib/test/data/stats.txt
lib/python3.10/site-packages/egglib/test/data/example.gnl
lib/python3.10/site-packages/egglib/test/data/LYK.E2.cds
lib/python3.10/site-packages/egglib/test/data/LYK.cds
lib/python3.10/site-packages/egglib/test/data/LYK.E.cds
lib/python3.10/site-packages/egglib/test/data/example_E.sta
lib/python3.10/site-packages/egglib/test/data/example.sta
lib/python3.10/site-packages/egglib/test/data/ehh_data01.hap
lib/python3.10/site-packages/egglib/test/data/ehh_data02.hap
lib/python3.10/site-packages/egglib/test/data/bta12_cgu.hap
lib/python3.10/site-packages/egglib/test/data/ehh_data02.inp
lib/python3.10/site-packages/egglib/test/data/ehh_data01.inp
lib/python3.10/site-packages/egglib/test/data/bta12_cgu.inp
lib/python3.10/site-packages/egglib/wrappers/apps.conf
lib/python3.10/site-packages/egglib/test/data/human_fragment.bed
lib/python3.10/site-packages/egglib/test/data/ania.clu
lib/python3.10/site-packages/EggLib-3.3.3.dist-info/INSTALLER
lib/python3.10/site-packages/EggLib-3.3.3.dist-info/WHEEL
lib/python3.10/site-packages/EggLib-3.3.3.dist-info/COPYING
lib/python3.10/site-packages/EggLib-3.3.3.dist-info/RECORD
lib/python3.10/site-packages/EggLib-3.3.3.dist-info/LICENSE
lib/python3.10/site-packages/EggLib-3.3.3.dist-info/METADATA
lib/python3.10/site-packages/EggLib-3.3.3.dist-info/REQUESTED
lib/python3.10/site-packages/egglib/cli/__init__.py
lib/python3.10/site-packages/egglib/test/wrappers/__init__.py
lib/python3.10/site-packages/egglib/test/coalesce/__init__.py
lib/python3.10/site-packages/egglib/test/__init__.py
lib/python3.10/site-packages/egglib/test/stats/__init__.py
lib/python3.10/site-packages/egglib/test/tools/__init__.py
lib/python3.10/site-packages/egglib/test/base/__init__.py
lib/python3.10/site-packages/egglib/test/io/__init__.py
lib/python3.10/site-packages/egglib/config.py
lib/python3.10/site-packages/egglib/coalesce/__init__.py
lib/python3.10/site-packages/egglib/test/io/test_index.py
lib/python3.10/site-packages/egglib/stats/__init__.py
lib/python3.10/site-packages/egglib/tools/__init__.py
lib/python3.10/site-packages/egglib/wrappers/__init__.py
lib/python3.10/site-packages/egglib/__init__.py
lib/python3.10/site-packages/egglib/io/__init__.py
lib/python3.10/site-packages/egglib/wrappers/_run.py
lib/python3.10/site-packages/egglib/eggwrapper.py
lib/python3.10/site-packages/egglib/tools/_to_codons.py
lib/python3.10/site-packages/egglib/io/_genepop.py
lib/python3.10/site-packages/egglib/test/stats/test_baudry.py
lib/python3.10/site-packages/egglib/test/io/test_fasta.py
lib/python3.10/site-packages/egglib/test/tools/test_reading_frame.py
lib/python3.10/site-packages/egglib/test/wrappers/test_clustal.py
lib/python3.10/site-packages/egglib/test/io/test_genepop.py
lib/python3.10/site-packages/egglib/test/stats/test_site_diversity.py
lib/python3.10/site-packages/egglib/test/base/test_site.py
lib/python3.10/site-packages/egglib/test/stats/test_paralog_pi.py
lib/python3.10/site-packages/egglib/test/io/test_legacy.py
lib/python3.10/site-packages/egglib/test/coalesce/test_simulator.py
lib/python3.10/site-packages/egglib/test/tools/test_concat.py
lib/python3.10/site-packages/egglib/test/stats/test_ehh.py
lib/python3.10/site-packages/egglib/test/wrappers/test_nj.py
lib/python3.10/site-packages/egglib/io/_ms.py
lib/python3.10/site-packages/egglib/stats/_baudry.py
lib/python3.10/site-packages/egglib/stats/_paralog_pi.py
lib/python3.10/site-packages/egglib/io/_fasta.py
lib/python3.10/site-packages/egglib/cli/config.py
lib/python3.10/site-packages/egglib/cli/test.py
lib/python3.10/site-packages/egglib/wrappers/_nj.py
lib/python3.10/site-packages/egglib/test/stats/test_ld.py
lib/python3.10/site-packages/egglib/stats/_haplotypes.py
lib/python3.10/site-packages/egglib/tools/_concat.py
lib/python3.10/site-packages/egglib/test/io/test_gff3.py
lib/python3.10/site-packages/egglib/test/stats/test_coding_diversity.py
lib/python3.10/site-packages/egglib/wrappers/_clustal.py
lib/python3.10/site-packages/egglib/_site.py
lib/python3.10/site-packages/egglib/tools/_reading_frame.py
lib/python3.10/site-packages/egglib/test/stats/test_haplotypes.py
lib/python3.10/site-packages/egglib/test/tools/test_random.py
lib/python3.10/site-packages/egglib/test/stats/test_freq.py
lib/python3.10/site-packages/egglib/test/wrappers/test_muscle.py
lib/python3.10/site-packages/egglib/stats/_ld.py
lib/python3.10/site-packages/egglib/wrappers/_utils.py
lib/python3.10/site-packages/egglib/test/tools/test_seq_manip.py
lib/python3.10/site-packages/egglib/test/io/test_ms.py
lib/python3.10/site-packages/egglib/test/wrappers/test_codeml.py
lib/python3.10/site-packages/egglib/io/_legacy.py
lib/python3.10/site-packages/egglib/stats/_coding_diversity.py
lib/python3.10/site-packages/egglib/test/wrappers/test_phyml.py
lib/python3.10/site-packages/egglib/alphabets.py
lib/python3.10/site-packages/egglib/tools/_seq_manip.py
lib/python3.10/site-packages/egglib/wrappers/_muscle.py
lib/python3.10/site-packages/egglib/_freq.py
lib/python3.10/site-packages/egglib/test/io/test_vcf.py
lib/python3.10/site-packages/egglib/test/io/test_genbank.py
lib/python3.10/site-packages/egglib/stats/_ehh.py
lib/python3.10/site-packages/egglib/coalesce/_simulator.py
lib/python3.10/site-packages/egglib/test/tools/test_code_tools.py
lib/python3.10/site-packages/egglib/test/coalesce/test_param_helpers.py
lib/python3.10/site-packages/egglib/wrappers/_phyml.py
lib/python3.10/site-packages/egglib/test/base/test_tree.py
lib/python3.10/site-packages/egglib/io/_gff3.py
lib/python3.10/site-packages/egglib/wrappers/_codeml.py
lib/python3.10/site-packages/egglib/io/_genbank.py
lib/python3.10/site-packages/egglib/tools/_code_tools.py
lib/python3.10/site-packages/egglib/test/base/test_structure.py
lib/python3.10/site-packages/egglib/io/_vcf.py
lib/python3.10/site-packages/egglib/stats/_cstats.py
lib/python3.10/site-packages/egglib/coalesce/_param_helpers.py
lib/python3.10/site-packages/egglib/test/io/test_vcf_legacy.py
lib/python3.10/site-packages/egglib/_tree.py
lib/python3.10/site-packages/egglib/wrappers/_blast.py
lib/python3.10/site-packages/egglib/test/base/test_alphabet.py
lib/python3.10/site-packages/egglib/test/wrappers/test_blast.py
lib/python3.10/site-packages/egglib/test/base/test_base.py
lib/python3.10/site-packages/egglib/test/stats/test_cstats.py
lib/python3.10/site-packages/egglib/_interface.py
lib/python3.10/site-packages/egglib/test/data/MtChr2_315000-415000.gb
lib/python3.10/site-packages/egglib/test/data/mask_nucl_windowmasker.asnb
lib/python3.10/site-packages/egglib/test/data/mask_nucl_dustmasker.asnb
lib/python3.10/site-packages/egglib/test/data/C94.gpop
lib/python3.10/site-packages/egglib/test/data/P93.gpop
lib/python3.10/site-packages/egglib/test/data/C94MD.gpop
lib/python3.10/site-packages/egglib/test/data/P93MD.gpop
lib/python3.10/site-packages/egglib/test/data/C98.gpop
lib/python3.10/site-packages/egglib/test/data/C98MD.gpop
lib/python3.10/site-packages/egglib/test/data/W08.gpop
lib/python3.10/site-packages/egglib/test/data/W08MD.gpop
lib/python3.10/site-packages/egglib/test/data/b.gff3
lib/python3.10/site-packages/egglib/test/data/a.gff3
lib/python3.10/site-packages/egglib/test/data/Populus_trichocarpa_short_thinned.gff3
lib/python3.10/site-packages/egglib/test/data/Osativa_193_gene_thinned.gff3
lib/python3.10/site-packages/egglib/test/data/gencode.v43.annotation_head.gff3
lib/python3.10/site-packages/egglib/libegglib.cpython-310-aarch64-linux-gnu.so
lib/python3.10/site-packages/egglib/random.cpython-310-aarch64-linux-gnu.so
lib/python3.10/site-packages/egglib/_eggwrapper.cpython-310-aarch64-linux-gnu.so
lib/python3.10/site-packages/egglib/test/data/b.bcf
bin/egglib-test
bin/egglib-config

mgrigorov in 🌐 euler-arm-22 in /tmp/egglib via 🅒 test123 took 3s 
❯ file lib/python3.10/site-packages/egglib/libegglib.cpython-310-aarch64-linux-gnu.so                                                                                                                      (test123) 
lib/python3.10/site-packages/egglib/libegglib.cpython-310-aarch64-linux-gnu.so: ELF 64-bit LSB shared object, ARM aarch64, version 1 (SYSV), dynamically linked, with debug_info, not stripped
```
LGTM!

**Comment by martin-g** [comment]
@BiocondaBot please add label

File: recipes/egglib/meta.yaml [hunk 1]
```diff
@@ -51,6 +51,9 @@ about:
   dev_url: "https://gitlab.com/demita/egglib/"
 
 extra:
+  additional-platforms:
+    - linux-aarch64
+    - osx-arm64
```
Comment (aliciaaevans) [inline recipes/egglib/meta.yaml:56]:
```suggestion
```
Probably best to remove this to avoid confusion since osx is skipped.

**Review by aliciaaevans — APPROVED** [review]

**Pull request merged**